spec/example_app/app/dashboards/order_dashboard.rb

```diff
@@ -14,6 +14,7 @@ class OrderDashboard < Administrate::BaseDashboard
     line_items: Field::HasMany,
     total_price: Field::Number.with_options(prefix: "$", decimals: 2),
     shipped_at: Field::DateTime,
+    status: Field::Enum.with_options(enum: {'pending' => 0, 'confirmed' => 1}),
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.